Electron/src/RAM/Science/components/Spectrometer.tsx

```diff
@@ -155,7 +155,6 @@ class Spectrometer extends Component<IProps, IState> {
   }
 
   getControl(): void {
-    rovecomm.resubscribe()
```

Comment:
i'm confused why this line was deleted?

Reply:
rovecomm.resubscribe is normally only called by pushing the button in RED when you first connect to the rover. We called it from getControl simply to remove that button for aesthetic purposes during a presentation. The rover needs to be subscribed to the science sensor board in order to get back valid data, but the resubscription should be handled where it is for every other board rather than resubscribing to all boards here just to make sure that we are subscribed to the science sensor board.